guides/common/modules/proc_customizing-registration-by-using-snippets.adoc

```diff
@@ -0,0 +1,23 @@
+[id="customizing-registration-by-using-snippets"]
+= Customizing registration by using snippets
+
+You can customize the registration process without editing the *Global Registration* template by creating snippets that the template includes if they exist.
+To add custom steps to registration, create one or both of the following snippets:
+
+* `before_registration` {endash} This snippet will be loaded and executed by the *Global Registration* template at the beginning of registration.
+* `after_registration` {endash} This snippet will be loaded and executed by the *Global Registration* template at the end of registration.
+
+You have to name the snippets precisely.
+Otherwise, they cannot be recognized.
+
+.Prerequisites
+* Your {Project} account has a role that grants the `view_provisioning_templates` and `create_provisioning_templates` permissions.
+* You have to select a particular organization and location context.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Templates* > *Provisioning Templates*.
+. Click *Create Template*.
+. In the *Name* field, enter the name of the required snippet.
+. In the template editor, create your snippet.
```

Comment:
Maybe mention the exact names again?

Reply:
Or how about moving the whole information about snippets to this step?

```
. In the *Name* field, enter the name of the required snippet:
.. To create a snippet that is loaded before registration:
.. To create a snippet that is loaded after registration:
```

I'm suggesting this because it's a good idea to mention the names again (as Maximilian suggests), but on the other hand, that would introduce duplication. And users will need to know how to name the snippets when following the steps, so if we want to avoid duplication, the procedure would be a better fit.

Reply:
I'll just mention the names.